camaragon/adcellerant-skills-fe
# Cma/feature/filter functionality

## Pull Request

<br>

### Description

* Is this a feature or a fix?

Feature

* Where should the reviewer start?

Start at the `Form.js` file.

* What functionalities do these changes effect? Why were these changes made? What was the motivation behind these changes?

These changes allow for the user to filter the analytics data in several different ways. They can do it individually or all at the same time. This makes it easier to pinpoint a specific section of the data.

* What outside features or research did you use to implement this fix?

I researched the `Date.parse()` command and some other date formatting tools.

* How should this be tested?

Run on browser and use the inputs to see if data is filtered properly.

![Screen Shot 2021-06-21 at 8 57 23 PM](https://user-images.githubusercontent.com/69489633/122855970-499eac00-d2d3-11eb-9427-4fce20543165.png)

<br>

***

#### Please review considerations below:
